### DISCUSSION TITLE
New release against updated Splat

### DISCUSSION THREAD

**Discussion by Rivsoft** [reactiveui/ReactiveUI#2753] [General]
Hi ReactiveUI team !!

I just noticed that a new ReactiveUI version was released a few hours ago but this one was built against the version of Splat that doesnt include the fixes to System.ValueTuple.

Is there any chance to release a minor version that includes a new version of Splat with the correct dependencies ?

Thanks in advance and keep up the good work !!

Best regards,

Mariano

**Comment by glennawatson** [comment]
Did you guys submit a splat PR as well?

**Reply by Rivsoft** [reply]
Yes, it was merged already too. https://github.com/reactiveui/splat/commit/82d3cb632c4bd31e455fbff5fb3244df957ad51a

**Comment by glennawatson** [comment]
Alright thanks. Will do a splat release then point release tomorrow.

**Reply by Rivsoft** [reply]
Thank you sir !!

**Comment by glennawatson** [comment]
All released

**Reply by Rivsoft** [reply]
Thank you sir !!
